test/artefacts/marriage-abroad/madagascar/ceremony_country/partner_british/opposite_sex.txt

```diff
@@ -0,0 +1,8 @@
+Marriage in Madagascar
```

Comment:
Are these files meant to be in here? Madagascar, and Malawi also further down.

Reply:
This is the result of running `bin/prep_for_pull_request.sh marriage-abroad`. It was probably missed in a previous PR.